docs/imageproxy.rst

```diff
@@ -0,0 +1,140 @@
+===========
+Image Proxy
+===========
+
+.. contents::
+
+The image proxy is an independent service to manipulate images in various ways
+(e.g. crop or resize images).
+
+The commands and the image URL have to be provided as query parameters and the
+whole query string has to be signed to disallow others to penetrate the service.
+
+
+Base URLs
+=========
+
+Each environment has it's own image proxy.
+
+On DEV environment::
+
+    https://imageproxy-iris-dev.lovelysystems.com
+
+On STAGING environment::
+
+    https://imageproxy-iris-stg.lovelysystems.com
+
+On PRODUCTION environment::
+
+    https://images.petition.ch
+
+
+Query Parameters
+================
+
+The general parameters are described in the `Pilbox Documentation
+<https://github.com/agschwender/pilbox#general-parameters>`_.
+
+Additional information:
+
+- `url`: The url of the image to be resized. The service is configured to use
+  the S3 base URL by default. For images uploaded by the :doc:`files` you
+  can skip the hostname and only use the file's ID with a leading slash.
+  It's also possible to use images from other locations by providing the
+  full URL.
+- `client`: The client parameter is required and must contain the value
+  ``iris``.
+
+
+Signature
+=========
+
+After creating a command with the query parameters, the request has to be signed
+with an additional query parameter ``sig``. This is a HMac-SHA1-hashed value
+of the previous part of the query parameter.
+
+All parts after question mark ``?`` have to be used as the ``message`` for the
+signing::
+
+    url=/ba1deb533c324cadbe4a8187fed2b5a2&w=200&h=200&mode=crop&client=iris
+
+The ``passphrase`` is the secret key for signing the request. This should not
```

Comment:
isn't "should not" too weak?

How about:
"The secret key must remain secret. Signed URLs have to be generated backend-side."